README.md

```diff
@@ -103,17 +103,22 @@ kubectl apply -f ./examples/kubernetes/demo-pod.yaml
 Setup GCP service account first and setup Kubernetes cluster
 ```
 $ ./deploy/project_setup.sh
-$ ./deploy/kubernetes/cluster_setup.sh
-```
-
-### Manual
-```
 # Build the Filestore CSI driver image and push to a container registry.
 $ PROJECT=your-gcp-project make build-image-and-push
-# Modify manifests under deploy/kubernetes/manifests to use development image
-$ ./deploy/kubernetes/driver_start.sh
+
+# The base manifests are listed under deploy/kubernetes/base.
+# The overlays (e.g prow-gke-release-staging-head, prow-gke-release-staging-rc, stable, noauth) are listed under deploy/kubernetes/overlays
+# For setup of the cluster with one of the overlays use:
+$ PROJECT=<your-gcp-project> GCFS_SA_DIR=<path-to-credentials-file> DEPLOY_VERSION=<overlay-name> ./deploy/kubernetes/cluster_setup.sh
+
+# 'noauth' overlay uses default service account for communicating with GCP services. Bring up GCE cluster with following:
+$ NODE_SCOPES=https://www.googleapis.com/auth/cloud-platform KUBE_GCE_NODE_SERVICE_ACCOUNT=<SERVICE_ACCOUNT_NAME>@$PROJECT.iam.gserviceaccount.com kubetest --up
+$ PROJECT=<your-gcp-project> DEPLOY_VERSION=noauth ./deploy/kubernetes/cluster_setup.sh
 ```
 
```

Comment:
Add here that the given node scopes will allow any pod to reach GCE as the service account, and so should only be used for testing and development, not production clusters.

Reply:
Done